modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase

```diff
@@ -167,12 +167,63 @@ definition {
 	}
 
 	@description = "LPS-135396 - Verify it is possible to add a Picklist field on layout"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanAddPicklistEntryOnLayout {
+		Picklist.gotoPicklists();
+
+		Picklist.addPicklistViaUI(picklistName = "Custom Picklist");
+
+		Picklist.gotoPicklists();
+
+		Picklist.gotoPicklistView(picklistName = "Custom Picklist");
+
+		Picklist.addPicklistItemViaUI(picklistItemName = "Item Test 1");
+
+		Picklist.clickSaveButton();
+
+		Picklist.gotoPicklists();
+
+		Picklist.gotoPicklistView(picklistName = "Custom Picklist");
+
+		Picklist.addPicklistItemViaUI(picklistItemName = "Item Test 2");
+
+		Picklist.clickSaveButton();
```

Comment:
This should be done via API. There are already created macros under `Picklist.macro`.